README.md

```diff
@@ -98,58 +97,42 @@ struct ChatView: View {
             if let atlassSwifUIView = AtlasSDK.getAtlassSwiftUIView() {
                 atlassSwifUIView
             } else {
-                Text("Unable to load chat. Ensure App ID is set.")
+                Text("Can not create AtlasSDK View")
             }
         }
     }
 }
 ```
 
----
+### Within UIKit
 
-### User Identification 
-
-The Atlas SDK allows the chat to operate without identification. In this case, conversations are initiated without associating them with a specific customer. However, developers can link conversations to existing customer by calling the `identify` method with a valid user ID.
+To display Atlas Chat using UIKit you need to get instance of the view controller and insert it into your view:
 
 ```swift
-// Identify a user before presenting the chat
-AtlasSDK.identify(userId: "UNIQUE_USER_IDENTIFIER", 
-                  userHash: nil, // Required if authenication is enabled in Atlas Configuration
-                  userName: "John Doe", // Optional
-                  userEmail: "john.doe@example.com") // Optional
-
-// Proceed to present the chat
-if let chatViewController = AtlasSDK.getAtlassViewController() {
-    navigationController?.present(chatViewController, animated: true)
+guard let atlassViewController = AtlasSDK.getAtlassViewController() else {
+    print("Can not create AtlasSDK View Controller")
+    return
 }
+  
+navigationController.present(atlassViewController, animated: true)
 ```
 
----
-
-### Customization 
+### Configuring view 
 
-It's possible to pass `query` key to a `getAtlassViewController` to configure the behavior or content of the returned AtlasFragment. (For ex: open specified chatbot to handle possible customer request)
-
-**Default value:** "" (empty string).
-
-**Expected format:** "key1: value1; key2: value2; ...."
+You can configure how does Atlas UI looks like on the [Chat Configuration page](https://app.atlas.so/configuration/chat). You can also configure the behavior of Atlas UI with the query parameter:
 
 ```swift
-// Initiate fragment and immediately start chatbot with report_bug key (chatbotKey: report_bug), or open the last one if exists (prefer: last)
-val atlasFragmentWithChatbot = AtlasSdk.getAtlasFragment(query = "chatbotKey: report_bug; prefer: last")
-// Initiate fragment and immediately open helpcenter
-val atlasFragmentWithHelpcenter = AtlasSdk.getAtlasFragment(query = "open: helpcenter")
-```
-
-**`chatbotKey: KEY`** Specifies the chatbot that has to be started immediately when AtlasFragment is loaded
+// Start chat with help center opened
+let atlassViewControllerWithHelpCenter = AtlasSDK.getAtlassViewController(query: "open: helpcenter")
```

Comment:
I find "Initialize" better the "Start". Since "Start" it when `viewDidApear` and webview start loading. In your example, you initializing the chat.